lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1473,6 +1476,27 @@ def cmd_sessions(*args)
       print_line
       print(Serializer::ReadableText.dump_sessions(framework, :show_extended => show_extended, :verbose => verbose, :search_term => search_term))
       print_line
+    when 'name'
+      if session_name.nil? || session_name.blank?
```

Comment:
Is just .blank sufficient here?  If nothing else, just for my own edification...  :P

Reply:
It used to be `session_name.strip.empty?`, so I forgot to remove the `nil` case. Thanks!

Reply:
(It's been hard for me to start adopting the Railsy stuff. :)